demo/src/ModelInput.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Button from './components/Button'
+import ModelIntro from './components/ModelIntro'
+
+
+// TODO: if you want to have some quickly-accessible examples, you can add them here.  You can then
+// auto-populate your input fields by selecting an example from the select box.
+
+const examples = [
+  {
+    field1: "field 1 input for example 1",
```

Comment:
I'm not sure how I feel about the names `field1` and `field2`, rather than something more descriptive like `question` and `passage` or whatever. In any case, we should probably give more explicit guidance about renaming them to the relevant inputs for your model

Reply:
I'll change them to `long_text_input` and `short_text_input`, similar to how I named things in `ModelOutput.jsx`.  Sound reasonable?

Reply:
👍